dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -1067,13 +1071,11 @@ public Scheduling validateScheduling(final Scheduling scheduling) {
         } else if(scheduling.startDate().isEmpty() && scheduling.endDate().isPresent()) {
             DotPreconditions.checkState(scheduling.endDate().get().isAfter(NOW),
                     "Invalid Scheduling. End date is in the past");
-            DotPreconditions.checkState(
-                    Instant.now().plus(EXPERIMENTS_MAX_DURATION.get(), ChronoUnit.DAYS)
-                            .isAfter(scheduling.endDate().get()),
-                    "Experiment duration must be less than "
-                            + EXPERIMENTS_MAX_DURATION.get() +" days. ");
 
-            toReturn = scheduling.withStartDate(Instant.now());
+            final Instant startDate = scheduling.endDate().get().minus(EXPERIMENTS_MAX_DURATION.get(),
```

Comment:
What if `scheduling.endDate()` is empty?

Reply:
it is the case above this one